### PR TITLE
1376

### PR DESCRIPTION
**Pull request by husseinalamutu**

### CODE REVIEW

**Review by chie2727 — COMMENTED** [review]
Hi @husseinalamutu ,
doesn't this function only need 1 variable to be returned?

**Comment by husseinalamutu** [comment]
> Hi @husseinalamutu , doesn't this function only need 1 variable to be returned?

Hi @chie2727 made some edit can you review it again

**Review by chie2727 — COMMENTED** [review]
In Line 15: `from ivy.backend_handler import current_backend as _cur_backend` ,

`current_backend` has been imported as `_cur_backend` so there is no need to make any change here.

**Comment by husseinalamutu** [comment]
> In Line 15: `from ivy.backend_handler import current_backend as _cur_backend` ,
> 
> `current_backend` has been imported as `_cur_backend` so there is no need to make any change here.
> 
> Could you send me a link to the issue you are working on? I think you might be editing the wrong file.

If that's the case, I don't think there is anymore reformating to be done. This is the link https://github.com/unifyai/ivy/issues/1376

**Comment by chie2727** [comment]
Hi @husseinalamutu ,
I'm assuming this is the task you're working on: https://github.com/unifyai/ivy/issues/791
I think you should leave a comment on that page by referencing your issue number #1376 in a task list.

As for the reformatting, the next steps are to: 3. add the correct docstrings, if necessary, and 4. add thorough docstring examples.
Follow the links to the contributor guide on the task page for more details.

**Comment by husseinalamutu** [comment]
Hi @chie2727 just made a merge request for to_list #791 #1376

**Comment by husseinalamutu** [comment]
> Hi @husseinalamutu , I'm assuming this is the task you're working on: #791 I think you should leave a comment on that page by referencing your issue number #1376 in a task list.
> 
> As for the reformatting, the next steps are to: 3. add the correct docstrings, if necessary, and 4. add thorough docstring examples. Follow the links to the contributor guide on the task page for more details.

Hi, I have done the reformating, do your review on it. Thanks.

File: ivy/functional/ivy/general.py
Comment (chie2727) [inline ivy/functional/ivy/general.py:635]: The ivy.Container inputs should be given as a keyword argument.

Take a look at this for example:
https://github.com/unifyai/ivy/blob/e13fe80e59240af64f8f16fadfcde1823ab0ac9d/ivy/functional/ivy/losses.py#L138

File: ivy/functional/ivy/general.py
Comment (chie2727) [inline ivy/functional/ivy/general.py:570]: Hi @husseinalamutu , thank you for making the edits!

Just a few more changes to make:

- [ ] 1. In line 570, please change `Examples` to `Functional Examples`.

- [ ] 2. Include a heading and section for `Instance Method Examples` for an ivy.Array instance method and an ivy.Container instance method.

- [ ] 3. Correct your usage of the ivy.Container input. (Have a look at my other comment for more detail.)

**Review by chie2727 — CHANGES_REQUESTED** [review]

**Comment by husseinalamutu** [comment]
I have made another commit, waiting for your review.

**Comment by husseinalamutu** [comment]
Hey @chie2727 still waiting for your feedback

File: ivy/functional/ivy/general.py
Comment (chie2727) [inline ivy/functional/ivy/general.py:694]: Hi @husseinalamutu ,
I just made some changes to the format of your output statements since they should be in the form `[-1, 0, 1]` and not `ivy.to_list([-1, 0, 1])` for example.

Last change to make: `ivy.NativeArray` doesn't work as an instance method here so you can remove the 3 examples at lines 694, 701 and 710, then replace these with just 1 example using only `ivy.Container` as an instance method.

Once you've made this change I can merge all your edits in.

**Review by chie2727 — CHANGES_REQUESTED** [review]

**Comment by husseinalamutu** [comment]
Hi, @chie2727 I would like you to enlighten me on how you could tell that ivy.NativeArray wouldn't work in the to_list case for ivy.Container instance method.

**Comment by chie2727** [comment]
Hi @husseinalamutu ,
You can verify this in your local environment.

For example:

clip_3

**Review by chie2727 — APPROVED** [review]

**Comment by chie2727** [comment]
Thanks for making all these edits! I'll merge your work in now

**Comment by husseinalamutu** [comment]
Thanks for the assistance, meanwhile I saw something earlier about the function formatting been updated, I think static examples was added. Don’t know if I should add that quickly so you can re merge it.

**Pull request merged**
